Makefile

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
my 2p is that the example could be in a README vs make, unless make is a commonly used tool in golang